src/analysis/retail/evoker/preservation/normalizers/CastLinkNormalizer.ts

```diff
@@ -593,6 +594,14 @@ const EVENT_LINKS: EventLink[] = [
     referencedEventType: EventType.Cast,
     backwardBufferMs: CAST_BUFFER_MS,
   },
+  {
+    linkRelation: STASIS_FOR_RAMP,
+    linkingEventId: SPELLS.STASIS_BUFF.id,
+    linkingEventType: EventType.RemoveBuff,
+    referencedEventId: TALENTS_EVOKER.EMERALD_COMMUNION_TALENT.id,
+    referencedEventType: EventType.ApplyBuff,
+    forwardBufferMs: MAX_ECHO_DURATION + 3000,
+  },
```

Comment:
Might want to add an additionalCondition(c) here for .hasTalents for Reso Sphere/Lifebind/whatever else here just in case